packages/appkit/tests/appkit.test.ts

```diff
@@ -869,6 +869,15 @@ describe('Base', () => {
     })
 
     it('should  sync balance correctly', async () => {
+      vi.spyOn(ChainController, 'state', 'get').mockReturnValue({
+        chains: new Map([['eip155', { namespace: 'eip155' }]]),
+        activeChain: 'eip155',
+        activeCaipNetwork: mainnet
+      } as any)
+      vi.spyOn(AccountController, 'state', 'get').mockReturnValue({
```

Comment:
Same for the account controller, calling syncAccount with mock user should authenticate the user